arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -1469,6 +1711,167 @@
+          },
+          "defaultPolicy":{
+            "$ref": "#/definitions/PolicyNameEnum",
+            "description": "Name of the Ssl predefined policy applied by default to application gateway"
+          },
+          "availableCipherSuites":{
+            "type":"array",
+            "items":{
+              "$ref": "#/definitions/CipherSuitesEnum"
+            },
+            "description": "List of available Ssl cipher suites."
+          },
+          "availableProtocols":{
+            "type":"array",
+            "items":{
+            "$ref": "#/definitions/ProtocolsEnum"
+          },
+          "description": "List of available Ssl protocols."
+        }
+      },
+      "description": "Properties of ApplicationGatewayAvailableSslOptions"
+    },
+    "ApplicationGatewayAvailableSslPredefinedPolicies": {
+      "properties": {
+        "value":{
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGatewaySslPredefinedPolicy"
+          },
+          "description": "List of available Ssl predefined policy."
+        }
+      },
+      "description": "Response for ApplicationGatewayAvailableSslOptions API service call."
+    },
+    "ApplicationGatewaySslPredefinedPolicy": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of Ssl predefined policy."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ApplicationGatewaySslPredefinedPolicyPropertiesFormat"
+        }
+      },
+      "allOf":[
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "An Ssl predefined policy"
+    },
+    "ApplicationGatewaySslPredefinedPolicyPropertiesFormat":{
+      "properties":{
+        "cipherSuites":{
+          "type":"array",
+          "items":{
+            "$ref": "#/definitions/CipherSuitesEnum"
+          },
+          "description": "Ssl cipher suites to be enabled in the specified order for application gateway."
+        },
+        "minProtocolVersion":{
+          "$ref": "#/definitions/ProtocolsEnum",
+          "description": "Minimum version of Ssl protocol to be supported on application gateway."
+        }
+      },
+      "description": "Properties of ApplicationGatewaySslPredefinedPolicy"
+    },
+    "PolicyNameEnum":{
+      "type": "string",
+      "enum":[
+        "AppGwSslPolicy20150501",
+        "AppGwSslPolicy20170401",
+        "AppGwSslPolicy20170401S"
+      ],
+      "x-ms-enum":{
+        "name": "ApplicationGatewaySslPolicyName",
+        "modelAsString": true
+      }
+    },
+    "ProtocolsEnum":{
+      "type": "string",
+      "enum":[
+        "TLSv1_0",
+        "TLSv1_1",
+        "TLSv1_2"
+      ],
+      "x-ms-enum":{
+        "name": "ApplicationGatewaySslProtocol",
+        "modelAsString": true
+      }
+    },
+    "CipherSuitesEnum":{
+      "type": "string",
```

Comment:
missing description.

Please add for better documentation. https://travis-ci.org/Azure/azure-rest-api-specs/jobs/241395215#L1579